### PR TITLE
Beacon tweak: removed redundant header for browse_signup.html

### PR DESCRIPTION
**Pull request by msknee**
#### What Changed

Header text for the subscribe form for the opportunities listing page, and individual opportunities pages was embedded in browse_signup.html, which made it impossible to title the form appropriately for its context. I simply took it out.
#### What's Needed

N/A
#### Screenshots

screen shot 2015-07-27 at 4 07 56 pm

(Contract shown is test data from existing RFP: http://apps.pittsburghpa.gov/omb/RFP_Final.pdf)

screen shot 2015-07-27 at 4 08 10 pm

### CODE REVIEW

File: purchasing/templates/opportunities/browse.html [hunk 1]
```diff
@@ -99,6 +99,8 @@
         {% endif %}
 
         {% if active|length > 0 or upcoming|length > 0 %}
+            <div class="spacer-20"></div>
```
Comment (bsmithgall) [inline purchasing/templates/opportunities/browse.html:114]: Fix the spacing here.

**Comment by bsmithgall** [comment]
I'm not really sure what you changed here. The purpose of the `browse_signup` file is to entirely contain that sign-up form, including the title. Can you maybe do a before/after screenshot?

File: purchasing/templates/opportunities/browse.html (hunk 1 above)
Comment (msknee) [inline purchasing/templates/opportunities/browse.html:114]: have it not be tabbed over? 
The spacer div is intentional.

**Comment by msknee** [comment]
screen shot 2015-07-27 at 5 44 23 pm

Before screenshot ^^ the title for the signup form doesn't work in the context of a single opportunity.

**Comment by bsmithgall** [comment]
Ah ok, that makes sense. Can you rename the file to something a bit more descriptive then? Maybe just like `opportunity_signup_form` or something like that. You'll also have to update the `include` references.

**Comment by bsmithgall** [comment]
:+1: thanks @shellicious

**Pull request merged**